pootle/runner.py

```diff
@@ -49,6 +49,14 @@ def init_settings(settings_filepath, template_filename):
     # We can't use regular python string formatting here.
     output = output.replace("${default_key}",
                             b64encode(os.urandom(KEY_LENGTH)).decode("utf-8"))
+    if db != "sqlite":
+        if db == "mysql":
+            db = "mysql"
+        elif db == "postgresql":
+            db = "postgresql_psycopg2"
+        output.replace("'ENGINE': 'transaction_hooks.backends.sqlite3'",
```

Comment:
I'd highly prefer not to replace an actual value, but a placeholder as we do with `SECRET_KEY` — less error-prone, no need to change values in multiple places. Also, `str.replace()` doesn't work in-place, it returns a modified copy.

```python
db_module = {
  'sqlite': 'sqlite3',
  'mysql': 'mysql',
  'postgresql': 'postgresql_psycopg2',
}[db]

output = output.replace('${db_module}', db_module)
```